### PR TITLE
Could use Metaspace to replace PermGen

### PR DESCRIPTION
**Pull request by yinlinchen**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -564,7 +564,7 @@
             This line enables code-coverage. However, it also blows the build's permgen.
             Fix me!
           -->
-          <argLine>-XX:-UseSplitVerifier ${jacoco.agent.unit.arg}</argLine>
+          <argLine>-XX:-UseSplitVerifier -XX:MaxMetaspaceSize=512m ${jacoco.agent.unit.arg}</argLine>
```
Comment (awoods) [inline pom.xml]: We can probably also remove `UseSplitVerifier`
http://stackoverflow.com/questions/31566672/java-8-what-is-the-equivalent-of-usesplitverifier

File: pom.xml [hunk 2]
```diff
@@ -545,7 +545,7 @@
       <plugin>
         <artifactId>maven-failsafe-plugin</artifactId>
         <configuration>
-          <argLine>${jacoco.agent.it.arg}</argLine>
+          <argLine>-XX:MaxMetaspaceSize=128m ${jacoco.agent.it.arg}</argLine>
```
Comment (awoods) [inline pom.xml]: Why `128m` here and `512m` below and `1024m` above?

Comment (yinlinchen) [inline pom.xml]: saw these from old pom.xml:  https://github.com/fcrepo4/fcrepo4/pull/883/files

**Comment by awoods** [comment]
Please create a JIRA ticket for tracking this update.

**Pull request merged**